src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  background-color: #e5e5e5;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.moyo-logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding: 23px 0;

Reply:
not fixed